packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -57,6 +57,7 @@ class CupertinoSwitch extends StatefulWidget {
     @required this.value,
     @required this.onChanged,
     this.activeColor,
+    this.dragStartBehavior = DragStartBehavior.start,
```

Comment:
What happens if this is set to null? Should there be an assert to ensure that nobody does that?